Sources/core/objects/Door.cpp

```diff
@@ -117,19 +117,19 @@ void Door::AttackBy(IdPtr<Item> item)
 {
     if (IdPtr<Weldingtool> w = item)
```

Comment:
This file is quite old, so codestyle here is not perfect. It will be nice at the same time with bugfix to fix short variables names here in `if` blocks. For example, here it will be better to name it `welder`. There are a couple more such places in the file.